EIPS/eip-4762.md

```diff
@@ -223,8 +202,8 @@ Reduce gas cost:
 |`WITNESS_BRANCH_COST`	|1900|
 |`WITNESS_CHUNK_COST`	|200|
 |`SUBTREE_EDIT_COST`	|3000|
-|`CHUNK_EDIT_COST`	|500|
-|`CHUNK_FILL_COST`	|6200|
+|`CHUNK_EDIT_COST`	    |500|
+|`CHUNK_FILL_COST`	    |6200|
```

Comment:
I can't add this comment in the intended line because there wasn't a change there.
I'm referring to lines L218-L220 and L226.

The intention in both places to say "skip it" is for gas cost charging. But the thing being skipped is also adding it to the underlying access list that are used for the witness creation. If those things also skip adding them to the list, then I think the witness wouldn't be correct?

Reply:
but the adding of stuff in the access list is done in previous sections, whereas this is specific to the charging of costs.